.pre-commit-config.yaml

```diff
@@ -1,38 +1,39 @@
-- repo: git://github.com/pre-commit/pre-commit-hooks
-  sha: master
-  hooks:
-  - id: trailing-whitespace
-  - id: flake8
-    args: [--max-line-length=120]
-  - id: check-merge-conflict
-  - id: double-quote-string-fixer
-  - id: end-of-file-fixer
-  - id: name-tests-test
-    args: [--django]
-  - id: debug-statements
-  - id: check-added-large-files
-  - id: check-ast
-  - id: check-byte-order-marker
-  - id: check-case-conflict
-  - id: check-docstring-first
-  - id: check-json
-  - id: pretty-format-json
-  - id: check-symlinks
-  - id: check-yaml
-  - id: detect-private-key
-  - id: requirements-txt-fixer
+-   repo: git://github.com/pre-commit/pre-commit-hooks
+    sha: v0.7.0
```

Comment:
@DrEdi no, using branches causes `pre-commit` do `git pull` every run. It will cause failures while running linters in vagga.